app/Filament/Resources/EggResource/Pages/CreateEgg.php

```diff
@@ -74,8 +74,7 @@ public function form(Form $form): Form
                                 ->helperText('')
                                 ->columnSpan(['default' => 1, 'sm' => 1, 'md' => 2, 'lg' => 2]),
                             TextInput::make('update_url')
-                                ->hintIcon('tabler-question-mark')
-                                ->hintToolTip('URLs must point directly to the raw .json file.')
+                                ->hintIcon('tabler-question-mark', tooltip: 'URLs must point directly to the raw .json file.')
```

Comment:
Yeh no... We're not doing it this way.

Reply:
No problem just wanted to fix the PHPstan error quickly.